packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md

```diff
@@ -81,13 +80,23 @@ Limits are applied on a per-recipient basis and vary by email type. The limit fo
 
 ### Per-user limits
 
-API endpoints that take username and password credentials, including the [Authentication API](/docs/reference/api/authn/) and the [OAuth 2.0 resource owner password flow](/docs/guides/implement-password/), have a per-username rate limit to prevent brute force attacks with the user's password:
+API endpoints that take username and password credentials, including the [Authentication API](/docs/reference/api/authn/), the [OAuth 2.0 resource owner password flow](/docs/guides/implement-password/), and SMS and Call factor endpoints, have a per-username rate limit to prevent brute force attacks with the user's password:
 
 | Action and Okta API endpoint                                      | Per User limits (all orgs) |
 | ----------------------------------------------------------------- | -------------------------: |
 | **Authenticate the same user:**<br>`/api/v1/authn`                | 4 per second               |
 | **Generate or refresh an OAuth 2.0 token:**<br>`/oauth2/v1/token` | 4 per second               |
 
+#### 30 second verification rate limits
+
+This rate limit applies to a user's attempt to send an SMS or Call enrollment or verification message to the same phone number.
+
+The Sign-In Widget works within this limit. It doesn't display the resend message button until 30 seconds have passed.
```

Comment:
You could drop this bit, unless we talk about the sign-in widget on this page.  I included it to make it clearer to you, but I don't know if we talk about sign-in widget on this page.